### ISSUE TITLE
[SUGGESTION] Font size, image view, etc

### ISSUE THREAD

**Issue by noblic** [defold/editor2-issues#2769]
Hello,

I'd like to suggest some features as below. If there are duplications then sorry for that and please ignore it. 

1. Sizing font on the UI.
A feature of sizing font not just editor's font size but also assets's tree items, tab titles, and etc on the UI.
For example, I'm using 4K monitor and text on UI is pretty small but I couldn't find any feature to size them.

2. I'm not able to see png files even though I double clicked it. It will be useful if there is a feature viewing image files in the Defold UI.

3.  Replace in files/project/path.

4. SFTP connection and remote file editing. 
This feature will be very useful when developing a game with web session.

5. Setting a shortcut key.

But Defold is already truly nice. 
Thank you.

**Comment by selimanac** [comment]
>     1. Sizing font on the UI.
>        A feature of sizing font not just editor's font size but also assets's tree items, tab titles, and etc on the UI.
>        For example, I'm using 4K monitor and text on UI is pretty small but I couldn't find any feature to size them.

Did you try [Distance field fonts](https://defold.com/manuals/font/?q=font#_distance-field-fonts) ?

**Comment by noblic** [comment]
No, I mean 'UI of Defold' not game made by Defold. I meant the development environment.

**Comment by noblic** [comment]
No, I mean 'UI of Defold' not game made by Defold. I meant the development environment.
